packages/front-end/components/Experiment/AnalysisForm.tsx

```diff
@@ -126,6 +174,12 @@ const AnalysisForm: FC<{
         if (experiment.status === "stopped") {
           body.phaseEndDate = dateEnded;
         }
+        if (usingSequentialTestingDefault) {
+          body.sequentialTestingEnabled = !!settings.sequentialTestingEnabled;
+          body.sequentialTestingTuningParameter =
+            settings.sequentialTestingTuningParameter ??
+            DEFAULT_SEQUENTIAL_TESTING_TUNING_PARAMETER;
+        }
```

Comment:
This logic looks a little weird.  Can you add a comment here about why we're ignoring the values entered in the form and always using org settings instead?

Reply:
This snippet is wrapped in:
`if (usingSequentialTestingDefault) {`
which means the user has explicitly checked to use the org default instead of the form settings. But I can add a comment.